deal_finder/constants.py

```diff
@@ -0,0 +1 @@
+INTERNAL_EMAILS = []
```

Comment:
What is this variable for ? 
It can be a more descriptive variable name

Reply:
This variable represents a list of email ids which would be used to notify on different occasions and get notified about the status of the different things when it is deployed (Like when it encounters an unexpected exception).\
Still thinking of a better name...

Reply:
Renamed it to `ERROR_NOTIFICATION_EMAIL_ADDRESSES`